src/js/media-container.js

```diff
@@ -24,7 +24,8 @@ template.innerHTML = `
       flex-direction: column-reverse;
 
       /* Default dimensions */
-      width: 720px;
+      width: 100%;
+      max-width: 720px;
```

Comment:
I'm a little worried about the effects of this on bigger videos, but we can leave it in and see.